_content/excerpts/can-monks-practice-astrology_kotyk-j.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Can Monks Practice Astrology? Astrology and the Vinaya in China."
+authors:
+  -  "Jeffrey Kotyk"
+external_url: "https://www.academia.edu/34408763/_Can_Monks_Practice_Astrology_Astrology_and_the_Vinaya_in_China_"
+drive_links:
+  - "https://drive.google.com/file/d/1FEAJAhNQwGIgvO1dYEY1csVcPJBLxnJh/view?usp=drive_link"
+course: medieval
+tags:
+  - astrology
+  - east-asian
+  - vinaya
```

Comment:
vinaya is not a tag. You have to choose between `vinaya-pitaka` (the canonical texts) and `vinaya-studies` (how the rules are interpreted and applied)